guides/rails_guides/markdown/renderer.rb

```diff
@@ -90,6 +90,7 @@ def convert_notes(body)
           end
         end
 
+        # rubocop:disable Lint/EndAlignment
```

Comment:
We don't want to start sprinkling these into our code base. The only allowed action is to fix our  rubocop rules.